LICENSE

```diff
@@ -0,0 +1,24 @@
+Copyright (c) 2015, The Regents of the University of California
```

Comment:
Huh? This code is not "Copyright (c) 2015, The Regents of the University of California", it's (c) 2018 by Me.

And I prefer to license the code under MIT.

Reply:
I will do that. thx for heads up.